internal/btpcli/facade_services_broker.go

```diff
@@ -47,3 +48,54 @@ func (f servicesBrokerFacade) GetByName(ctx context.Context, subaccountId string
 		"name":       brokerName,
 	}))
 }
+
+type SubaccountServiceBrokerRegisterInput struct {
+	Subaccount  string              `btpcli:"subaccount"`
+	Name        string              `btpcli:"name"`
+	Description string              `btpcli:"description"`
+	User        string              `btpcli:"user"`
+	Password    string              `btpcli:"password"`
+	URL         string              `btpcli:"url"`
+	Labels      map[string][]string `btpcli:"labels"`
+}
+
+func (f servicesBrokerFacade) Register(ctx context.Context, args SubaccountServiceBrokerRegisterInput) (servicemanager.ServiceBrokerResponseObject, CommandResponse, error) {
+	params, err := tfutils.ToBTPCLIParamsMap(args)
+
+	if err != nil {
+		return servicemanager.ServiceBrokerResponseObject{}, CommandResponse{}, err
+	}
+
+	return doExecute[servicemanager.ServiceBrokerResponseObject](f.cliClient, ctx, NewRegisterRequest(f.getCommand(), params))
+}
+
+type SubaccountServiceBrokerUpdateInput struct {
+	Id          string              `btpcli:"id"`
+	Subaccount  string              `btpcli:"subaccount"`
+	NewName     string              `btpcli:"newName"`
+	Description string              `btpcli:"description"`
+	User        string              `btpcli:"user"`
+	Password    string              `btpcli:"password"`
+	URL         string              `btpcli:"url"`
+	Labels      map[string][]string `btpcli:"labels"`
+}
+
+func (f servicesBrokerFacade) Update(ctx context.Context, args SubaccountServiceBrokerUpdateInput) (servicemanager.ServiceBrokerResponseObject, CommandResponse, error) {
+	params, err := tfutils.ToBTPCLIParamsMap(args)
+
+	if err != nil {
+		return servicemanager.ServiceBrokerResponseObject{}, CommandResponse{}, err
+	}
+
+	return doExecute[servicemanager.ServiceBrokerResponseObject](f.cliClient, ctx, NewUpdateRequest(f.getCommand(), params))
+}
+
+// TODO Update
```

Comment:
The `Update` function is available. Does this mean something different?

Reply:
Good catch. I'll remove it.